dallinger/mturk.py

```diff
@@ -374,7 +405,7 @@ def revoke_qualification(self, qualification_id, worker_id, reason=""):
             )
         )
 
-    def get_qualification_score(self, qualification_id, worker_id):
+    def current_qualification_score(self, qualification_id, worker_id):
```

Comment:
I guess this is as good a time as any to rename things, but I'm not sure I follow the distinction being made here.

Reply:
Seemed critical at the time for some reason, but in retrospect it doesn't seem to make much of a difference.